docs/src/gui/qtvcp.adoc

```diff
@@ -277,12 +277,12 @@ Here we show how to add new functions and override existing ones.
 # reference: https://ruivieira.dev/python-monkey-patching-for-readability.html
 import types
 
-# import the handlerfile to get reference to it's libraries.
+# import the handlerfile to get reference to its libraries.
 # use <screenname>_handler
 import qtdragon_handler as hdlr
 
 # This is actually an unbounded function with 'obj' as a parameter.
-# You call this function without the usual preceding 'self.'
+# You call this function without the usual preceding 'self.'.
```

Comment:
I suspect this should read "'self'.", without the extra dot.

Reply:
;)

The final dot is the end of that sentence. The dot embraced by the ''s is meant to reference the use of self with an attribute name, as in 'self.whatever', see https://github.com/LinuxCNC/linuxcnc/pull/2797#discussion_r1430644909 .

Reply:
I tend to the version with the dot at the end as it marks the end of the sentence.